docs/management/admin/response-actions.asciidoc

```diff
@@ -105,20 +108,20 @@ TIP: You can also get a list of commands in the <<help-panel,Help panel>>, which
 
 Click image:images/help-icon.png[Help icon,17,18] *Help* in the upper-right to open the *Help* panel, which lists available response action commands and parameters as a reference.
 
+[role="screenshot"]
+image::images/response-console-help-panel.png[Help panel,50%]
+
 You can use this panel to build commands with less typing. Click the add icon (image:images/add-command-icon.png[Add icon,17,17]) to add a command to the input area, enter any additional parameters or a comment, then press *Return* to run the command.
 
+If the endpoint is running an older version of {agent}, some response actions may not be supported, as indicated by an informational icon and tooltip. {fleet-guide}/upgrade-elastic-agent.html[Upgrade {agent}] on the endpoint to add support for the latest response actions.
```

Comment:
@joepeeples while this will be true in `8.6`, this functionality was not released in `8.5`.  If the user has an Endpoint that is older than `8.4` the Responder option in the menu will be disabled with a message to upgrade.

Here is a screenshot of what users will see in `8.5`.

<img width="1726" alt="image" src="https://user-images.githubusercontent.com/56395104/195708983-f8c0f7a2-a8b6-48f9-bc46-b95aaff373a4.png">

Reply:
Note (mostly) to self: I'll revise this section and screenshot in this PR, which will merge to `main` (aka 8.6.0) and backport to `v8.5.0`. I also created docs issue https://github.com/elastic/security-docs/issues/2575 for 8.6.0, so I can update `main`/8.6.0 separately. 

There's probably some kind of Git wizardry to do this all in one swoop, but opting to keep it simple. Also AFAIK the feature hasn't actually been re-enabled yet, per https://github.com/elastic/security-team/issues/5161, so I can follow up on my docs issue once the feature itself has merged.